examples/tree_1d_dgsem/elixir_navierstokes_convergence_walls.jl

```diff
@@ -121,9 +121,10 @@ end
 initial_condition = initial_condition_navier_stokes_convergence_test
 
 # BC types
-velocity_bc_left_right = NoSlip((x, t, equations) -> initial_condition_navier_stokes_convergence_test(x,
-                                                                                                      t,
-                                                                                                      equations)[2])
+velocity_bc_left_right = NoSlip() do x, t, equations
+    u_cons = initial_condition_navier_stokes_convergence_test(x, t, equations)
+    return u_cons[2] / u_cons[1]
```

Comment:
```suggestion
    return SVector(u_cons[2] / u_cons[1])
```
Shouldn't the 1D version also return an `SVector`? Would this explain the allocations seen in #2173 ?

Reply:
Actually not, as the other entries in `flux` are also only scalars:

https://github.com/trixi-framework/Trixi.jl/blob/21a5a8d27042aa852ed421928d0617eb8744092f/src/solvers/dgsem_tree/dg_1d_parabolic.jl#L322-L329

Reply:
For the 2D cases, this could also be simply a tuple:

https://github.com/trixi-framework/Trixi.jl/blob/21a5a8d27042aa852ed421928d0617eb8744092f/src/equations/compressible_navier_stokes_2d.jl#L307-L310

Reply:
Ah, okay. Then this looks good from my side.